### ISSUE TITLE
Allow setting up biometric authentication without the use of a PIN

### ISSUE THREAD

**Issue by Etim-Orb** [vector-im/element-android#6830]
#### What would you like to do?
Do not request a PIN for setting up biometric authentication. 
#### Why would you like to do it?
So people can use extra authentication layers without the need of extra (potentially unneeded) credentials to remember.
#### How would you like to achieve it?
Allow setting up both a PIN and/or a Biometric auth but do not require one of them to achieve the other.

### Are you willing to provide a PR?

No

**Comment by ouchadam** [comment]
I believe there was a recent change to require pin unlock if the biometrics change, eg a new fingerprint added. It also acts as a fallback for if all biometrics are removed, the app is still pin protected.  

cc @jmartinesp

**Comment by jmartinesp** [comment]
Yes, biometric authentication is a 'shortcut' to avoid writing your pin code every single time, but it does need the underlying pin code as a fallback measure. I think that's pretty standard in most of the fingerprint protected apps I've used.

**Comment by Etim-Orb** [comment]
Signal/Molly, Infinity for Reddit, Bitwarden, Aegis, Tutanota, Simplelogin, Anonaddy, does not use an additional pin. They either use the device pin as fallback, no fallback or use their own registered necessary authentication such as passwords.
